ci-operator/config/3scale/3scale-operator/3scale-3scale-operator-3scale-2.14-stable.yaml

```diff
@@ -0,0 +1,53 @@
+binary_build_commands: make manager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.8"
```

Comment:
OCP 4.8? what is the minimum version supported by 3scale 2.14?

Reply:
Thank you for comment. Updated to 4.12